plugin-dotnet-agent/src/main/kotlin/jetbrains/buildServer/runners/CommandLineResult.kt

```diff
@@ -0,0 +1,15 @@
+package jetbrains.buildServer.runners
+
+import org.apache.xpath.operations.Bool
+
+data class CommandLineResult(
+        private val _exitCode: Sequence<Int>,
```

Comment:
The number of exit codes of command result could be 0 or 1? Do you have any idea which type we could use - nullable is not good from my point if view